packages/desktop-gui/src/specs/specs-list.jsx

```diff
@@ -110,12 +114,17 @@ class SpecsList extends Component {
             <input
               id='filter'
               className='filter'
-              placeholder='Search...'
+              placeholder={this._togglePlaceholderSearchTips()}
               value={specsStore.filter || ''}
               ref={this.filterRef}
+              onBlur={this._setBlur}
               onChange={this._updateFilter}
+              onFocus={this._setFocus}
               onKeyUp={this._executeFilterAction}
             />
+
+            { window.addEventListener('keydown', this._executeSearch) }
```

Comment:
The function is focusing the input when search keys are detected. Think this naming is a bit clearer of the function.

```suggestion
            { window.addEventListener('keydown', this._focusWhenSearchKeys) }
```

Reply:
@jennifer-shehane I have addressed the comment & pushed the changes.